frontend/src/employee-mobile-frontend/child-attendance/actions/MarkDeparted.tsx

```diff
@@ -101,7 +100,11 @@ export default React.memo(function MarkDeparted() {
     AbsenceTypeWithNoAbsence | undefined
   >(undefined)
 
-  const groupNotes = useQueryResult(groupNotesQuery(groupId))
+  const groupId = child.map(({ groupId }) => groupId).getOrElse(null)
+  const rawGroupNotes = useQueryResult(groupNotesQuery(groupId ?? ''), {
+    enabled: !!groupId
+  })
+  const groupNotes = groupId ? rawGroupNotes : Success.of([])
```

Comment:
This place would benefit from something like `constantQuery([])`, be we don't have that yet.